.github/workflows/master.yml

```
@@ -1,13 +1,10 @@
-# This workflow will build a .NET project
-# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-net
-
+# Protected-branch and scheduled CI for master. Token-bearing analysis stays
+# here, away from pull request jobs that execute PR-controlled code.
 name: master
 
 on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)